SConstruct

```diff
@@ -1021,43 +1021,47 @@ env['python_cmd_esc'] = quoted(env['python_cmd'])
 
 # Python 2 Package Settings
 cython_min_version = LooseVersion('0.23')
+numpy_min_version = LooseVersion('1.6.1')
```

Comment:
I agree that treating this like Python 3.2 support would be reasonable, but that's not what this commit does -- it aborts compilation of the Python module if an older Numpy is found. Printing the installed Numpy version in the build log is definitely useful, and if you want to print a warning if that version is older than what we test on, that would be fine too, but I don't want to force anyone to upgrade just because we haven't tested on an old version of Numpy that might work just fine.